### PR TITLE
Added Process > Determine audience

### PR DESCRIPTION
**Pull request by joshuacc**
Anyone have time to do some proofreading on this? :pencil:

### CODE REVIEW

File: manuscript/process.md [hunk 1]
```diff
@@ -79,4 +79,70 @@ With this more robust goal in mind we're more likely to communicate,
...
+### Who will be reading this?
+
+Often the readership of documents is wider than we would initially expect.
+A framework's website will be read by developers, of course,
+but perhaps also:
+
+* Architects evaluating technology choices
+* Managers trying to keep abreast of what their team is using
+* Technical recruiters trying to understand the skillset they are hiring for
+* Designers trying to understand technical constraints on their designs
```
Comment (ghost) [inline manuscript/process.md:97]: _of_ their designs

File: manuscript/process.md [hunk 2]
```diff
@@ -79,4 +79,70 @@ With this more robust goal in mind we're more likely to communicate,
...
+In most cases there will be a single group of readers who constitute your primary audience.
+
+Continuing the previous example:
+the primary audience of a framework website is developers.
+Others are important, but secondary to that audience.
+
+### What is their relation to the subject?
+
+At this point we have our primary audience,
+but haven't given much dtail about them.
```
Comment (ghost) [inline manuscript/process.md]: d_e_tail

**Comment by ghost** [comment]
That's all I found, the rest looks good (not a native speaker though)

:100:

File: manuscript/process.md (hunk 1 above)
Comment (ghost) [inline manuscript/process.md:97]: Oh I think _on_ is correct, since the design is not the origin of the constraints.

**Comment by joshuacc** [comment]
Thanks for the proofreading, @lgierth !

**Pull request merged**